test/btc-transaction-helper.test.js

```diff
@@ -0,0 +1,42 @@
+const { assert } = require('chai');
+const BtcTransactionHelper = require('../index').BtcTransactionHelper;
+const network = require('../index').networks.regtest;
+const bitcoinJsLib = require('bitcoinjs-lib')
+
+const config = {
+    host: 'localhost',
+    port: 18332,
+    user: 'rsk',
+    pass: 'rsk',
+    timeout: 30000,
+    network: network
+};
+
+describe('btc-transaction-helper', () => {
```

Comment:
I think it might be a better idea to move this into samples folder. We can create another story to find the way to create proper tests.

Having an integration test here would make github workflows fail if we integrate them into this repo